### PR TITLE
Update changelog and prow tag

### PR DESCRIPTION
**Pull request by hsadoyan**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by hsadoyan** [comment]
[1.8.2-rc1 Image](https://pantheon.corp.google.com/gcr/images/k8s-staging-cloud-provider-gcp/global/gcp-compute-persistent-disk-csi-driver@sha256:5d6dcb9a3b8bb6db7ffd4573a8f27b217bc6731b88f03470e2a7fc995dd92d94/details?jsmode=o&mods=logs_tg_prod)

File: CHANGELOG/CHANGELOG-1.8.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+# v1.8.2 - Changelog since v1.8.1
+
+
+## Changes by Kind
+
+### Other (Cleanup or Flake)
+
+- Update to go 1.19.4 ([#1103](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1103), [@msau42](https://github.com/msau42))
```
Comment (sunnylovestiramisu) [inline CHANGELOG/CHANGELOG-1.8.md:8]: Is this generated by the script? Seems like the change log does not include the cherry-picks prs: https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1115 is this expected for this repo's change-log?

Comment (leiyiz) [reply CHANGELOG/CHANGELOG-1.8.md:8]: all of the PRs cherry picked by #1115 does not have release notes, so it's normal that the changelog does not include it.

**Review by leiyiz — COMMENTED** [review]
/lgtm
/approve

**Comment by mattcary** [comment]
Very weird, the test grid is showing no jobs between the 21st and yesterday, and only one job yesterday.

https://testgrid.k8s.io/provider-gcp-compute-persistent-disk-csi-driver#presubmit-gcp-compute-persistent-disk-csi-driver-kubernetes-integration

Let me see if someone knows about this.

**Pull request merged**